packages/client/components/MassInvitationTokenLink.tsx

```diff
@@ -56,8 +56,12 @@ const MassInvitationTokenLink = (props: Props) => {
     }
     doFetch().catch()
   }, [])
+  const HOST = window.location.host
+  const isDevelopment = HOST != 'action.parabol.co'
```

Comment:
@mikennaji I think there are some changes I'd like to make to this, but I want to check in with @mattkrick 

Let's make the shortlink an env var: `SHORTLINK=prbl.in` If it's set we'll use it. If it isn't set we'll use `makeAppLink()`. This would sure make testing staging/dev easier :)

Ok with you?

Reply:
Sounds good ! :)

Reply:
yep! ENV var for the shortlink seems like the best path forward.